### DISCUSSION TITLE
이렇게 빨간 핑뜨면 안되는거죠..?

### DISCUSSION THREAD

**Discussion by hyojik123** [changdoc/naver-vaccine-macro#194] [Q&A]
설명대로 했는데 잘 안되는거 같아서요...
![image](https://user-images.githubusercontent.com/88326431/127959823-6465c642-2bdb-41e5-a074-ff58eadc1962.png)

**Comment by hyojik123** [comment]
![image](https://user-images.githubusercontent.com/88326431/127960173-d5fc3c63-dc21-4159-b646-eb6bc914262d.png)

**Comment by changdoc** [comment]
네 정상 동작 하지 않는 상황입니다.

https://github.com/changdoc/naver-vaccine-macro/discussions/177
이 이슈와 같은 상황 같네요. 아직 원인파악을 못 해 수정을 못하고 있습니다.
